service/src/DCGMIOGroup.hpp

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2015 - 2021, Intel Corporation
```

Comment:
```suggestion
 * Copyright (c) 2015 - 2022, Intel Corporation
```

This change went in for the rest of the files in the repo recently.  All of the license headers for this PR need to be updated, please.